src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -20,18 +20,29 @@ namespace System.Net.Http
     public partial class HttpClientHandler : HttpMessageHandler
     {
         private readonly HttpHandlerType _underlyingHandler;
-        private readonly DiagnosticsHandler? _diagnosticsHandler;
+
+#if TARGET_BROWSER
+        private readonly HttpMessageHandler _handler;
+#else
+        private HttpMessageHandler _handler => _underlyingHandler;
```

Comment:
It's _really_ strange to have a property named `_handler`.  How about:
```C#
private HttpMessageHandler Handler
#if TARGET_BROWSER
    { get; }
#else
    => _underlyingHandler;
#endif
```
or something along those lines?  And then use `Handler` elsewhere in the type.

Reply:
That's nicer, I'll change it